packages/genie/package.py

```diff
@@ -88,7 +88,7 @@ def url_for_version(self, version):
     patch("patch/sles-cnl.patch", when="platform=cray")
     patch("patch/root_subdir.patch")
 
-    patch("patch/GENIE-Generator.patch", when="@3.04.02")
+#    patch("patch/GENIE-Generator.patch", when="@3.04.02")
```

Comment:
Suggest removing the line entirely.

Reply:
Based on SciSoft discussion this line should be kept, but with `when="@3.04.00"`.